othello/settings/__init__.py

```diff
@@ -219,5 +235,12 @@
 
 if not DEBUG:
     sentry_sdk.init(
-        SENTRY_DSN, integrations=[DjangoIntegration(), CeleryIntegration()], send_default_pii=True,
+        SENTRY_DSN,
+        integrations=[DjangoIntegration(), CeleryIntegration()],
+        send_default_pii=True,
     )
+
+TESTING = "test" in sys.argv
+if TESTING:
+    DATABASES["default"]["ENGINE"] = "django.db.backends.sqlite3"
```

Comment:
I believe I added a note in README.md but the Game-related models use fields that are not support by sqlite. Dev environments should use postgresql either through Docker or Vagrant.